modules/page/html/render/message.js

```diff
@@ -229,15 +249,24 @@ exports.create = function (api) {
     api.sbot.async.get({ id, private: true, unbox }, (err, value) => {
       if (err) return cb(err)
       var msg = { key: id, value }
-      if (isBlog(msg)) {
-        Blog(api.sbot.obs.connection).async.get(msg, (err, result) => {
-          if (err) return cb(err)
-          msg.body = result.body
-          cb(null, msg)
+
+      var me = api.keys.sync.id()
+      onceTrue(api.sbot.obs.connection, sbot => {
+        sbot.patchwork.contacts.isBlocking({ source: me, dest: value.author }, (err, blocking) => {
+          if (err) throw err
```

Comment:
We usually do something like:

```js
if (err) return cb(err)
```

Do we need to throw here?

Reply:
I think I saw standard complain about this and I just put a throw in there by habit, still not very used to callbacks :smile:

Reply:
It says this conversation was resolved but I still think that we want to callback the error. Usually callbacks take `cb(err, val)` where `err` is either `null` or an error object and `val` is the value that you'd otherwise `return` if it wasn't an async callback function.

Reply:
I think you're looking at old code here, it's currently `if (err) return cb(err)`